terraform/cloud-platform-components/fluentd.tf

```diff
@@ -5,12 +5,12 @@ resource "helm_release" "fluentd_es" {
 
   set {
     name  = "fluent_elasticsearch_host"
-    value = "${terraform.workspace == local.live_workspace ? "search-cloud-platform-live-dibidbfud3uww3lpxnhj2jdws4.eu-west-2.es.amazonaws.com" : "search-test-yuq2c7ziybjvpmr2tllkghh6va.eu-west-2.es.amazonaws.com"}"
+    value = "${replace(terraform.workspace, "live", "") != terraform.workspace ? "search-cloud-platform-live-dibidbfud3uww3lpxnhj2jdws4.eu-west-2.es.amazonaws.com" : "search-test-yuq2c7ziybjvpmr2tllkghh6va.eu-west-2.es.amazonaws.com"}"
```

Comment:
Again with this, I think it's a bit hard to parse. Do we currently need to plan for other live clusters? I don't believe we plan to get any more anytime soon. Instead, the previous statement was easier to understand. Wdyt?

Reply:
we have live-0 and live-1 so

Reply:
I'm saying this because `live-0` is not managed from this branch and won't live long enough to try and integrate. Is `live-2` even a concern right now that would warrant complicating this?